src/components/templates/Poster/Poster.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faCog,
+  faShare,
+  faTv,
+  faStop,
+} from "@fortawesome/free-solid-svg-icons";
+import { OverlayTrigger, Popover } from "react-bootstrap";
+
+import { setVenueLiveStatus } from "api/venue";
+
+import { POSTER_CELL_COUNT_MAX } from "settings";
+
+import { PosterVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import { VideoParticipant } from "components/organisms/Video";
+import { UserList } from "components/molecules/UserList";
+import { Toggler } from "components/atoms/Toggler";
+
+import { usePosterVideo } from "./usePosterVideo";
+
+import { IntroVideoPreviewModal } from "./components/IntroVideoPreviewModal";
+
+import "./Poster.scss";
+
+export interface PosterProps {
+  venue: WithId<PosterVenue>;
+}
+
+export const Poster: React.FC<PosterProps> = ({ venue }) => {
+  const [isIntroVideoShown, setIntroVideoShown] = useState<boolean>(false);
+
+  const showIntroVideoModal = () => {
+    setIntroVideoShown(true);
+  };
+
+  const hideIntroVideoModal = () => {
+    setIntroVideoShown(false);
+  };
+
+  const {
+    activeParticipants,
+    passiveListerens,
+
+    isMeActiveParticipant,
+
+    turnVideoOff,
+    turnVideoOn,
+  } = usePosterVideo(venue.id);
+
+  const isPosterLive = venue.isLive;
+
+  const setVenueLiveOn = () => {
+    setVenueLiveStatus(venue.id, true);
+  };
+
+  const setVenueLiveOff = () => {
+    setVenueLiveStatus(venue.id, false);
+  };
+
+  const videoParticipants = activeParticipants.map((participant) => (
+    <VideoParticipant
+      participant={participant}
+      key={participant.identity}
+      additionalClassNames="poster__video-participant"
+    />
+  ));
+
+  const hasFreeSpace = videoParticipants.length < POSTER_CELL_COUNT_MAX;
+
+  return (
+    <>
+      <div className="poster">
+        <div className="poster__header">
+          <div />
+
+          <div className="poster__header--middle-cell">
+            <p className="poster__title">{venue.title}</p>
+            <div className="poster__categories" />
+          </div>
+
+          <div className="poster__header--right-cell">
+            {isMeActiveParticipant && (
+              <div
+                className="poster__control poster__control--stop"
+                onClick={turnVideoOff}
+              >
+                <FontAwesomeIcon icon={faStop} size="lg" />
+                <span className="poster__control-text">Stop video</span>
+              </div>
```

Comment:
Since this same pattern is used so often here, it would make sense to extract a simple little `PosterControl` component that wraps up this main class, icon, text, etc.

It should end up working something like:

```
<PosterControl
  containerClassNames="poster__control--stop"
  icon={faStop}
  label="Stop video"
  onClick={turnVideoOff}
/>
```

Then this should be used for the rest of the controls here as well.

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1299/commits/c23cde4297f7a38d803e0c4f837e99a9c9bdccdd